docs/guidelines/Composer-help-contents.md

```diff
@@ -0,0 +1,137 @@
+
+This document includes:
+
+- [Terminology](#terminology)
+- [Guidelines](#guidelines)
+  - [General guidelines](#general-guidelines)
+  - [Button names](#button-names)
+  - [Menu names](#menu-names)
+  - [Instructional copy](#instructional-copy)
+  - [Tooltips](#tooltips)
+  - [Error messages](#error-messages)
+  - [Watermark texts](#watermark-texts)
+  - [Help links](#help-links)
+- [Resources](#resources)
+
+# Terminology
+
+* **Instructional copy**: instructional texts that provide users with information about what to do.
+* **Tooltip**: texts displayed in an informational text box when hovering over an item.
+* **Watermark text**: text that lives inside an edit box or entry field.
+* **Help link**: context-sensitive help (CSH) link added to the user interface to help users with a concept or procedure while using a product.
+
+# Guidelines
+
+## General guidelines
+
+The general guidelines listed in this section apply to all types of help contents as documented in this doc.
+
+1. Be concise.
+2. Be syntactically accurate.
+3. Be contextually meaningful.
+4. Be consistent in language and style within the product and across the product portfolio (related products).
+5. Use sentence case unless texts contain proper names.
+
+## Button names
+
+1. Keep names short and limit to **3** words maximum.
+2. Where possible, reiterate the action to be performed upon click.
+3. Keep names common, clear, and concise.
+4. Should only contain letters.
+5. Don’t use acronyms.
+
+* **Example**:
+
+    ![button name example](./media/azure-button.png)
+
+## Menu names
+
+1. Keep names short and limit to **2** words maximum.
+2. Keep names common, clear, and concise.
+3. Should only contain letters.
+4. Don’t use acronyms.
+
+* **Example**:
+
+    ![menu name example](./media/azure-menu.png)
+
+## Instructional copy
+
+1. Limit to **1** phrase or **2** sentences in English.
+2. When necessary, include a help link pointing to a specific doc.
+3. Instructional text can be accompanied by illustrations or diagrams, below or next to the copy.
+4. Meet accessibility guidelines (including how they are invoked and presented).
+5. Meet Microsoft Fluent design guidelines.
+6. Don’t use acronyms.
+7. Don’t use questions.
+8. Don't repeat the label or name of the element.
+
+* **Example**:
+
+    ![instructional copy example](./media/pva-instructional-copy.png)
+
+## Tooltips
+
+1. Limit to 1 phrase or 2 sentences in English.
+2. Can include visuals or diagrams.
+3. Meets accessibility guidelines (including how they are invoked and presented).
+4. Meet Microsoft Fluent design guidelines.
+5. Meet accessibility guidelines (both mouse and keyboard hover).
+6. Do not replace, nor should be used as instructional copy.
```

Comment:
Can we provide some guidelines for what type of information is appropriate for tooltips versus what is appropriate for instruction copy? Otherwise, I think people may get them confused.

Reply:
Adding the following note in the doc:
> [!NOTE]
> Please distinguish between "instructional copy" and "tooltip":
> * An "instructional copy" provides the user with information about what to do rather than what a UX element or label is.
> * A "tooltip" provides the user with information about what a UX element or label is.